common/state/src/lib.rs

```diff
@@ -98,6 +98,12 @@ impl AgentState {
 #[derive(Hash, Clone, PartialEq, Eq)]
 pub struct FileName(bytes::Bytes);
 
+impl FileName {
+    pub fn bytes(&self) -> &[u8] {
```

Comment:
I think this is not used.

Reply:
I was using it to print out the contents. I've replaced it with derive(Debug) on the type